packages/shared/lib/core/profile/helpers/buildNewProfile.ts

```diff
@@ -31,6 +32,7 @@ export function buildNewProfile(
         settings: {
             currency: AvailableExchangeRates.USD,
             networkConfig: getOfficialNetworkConfig(networkProtocol, networkType),
```

Comment:
We should also try and remove the notion of `networkConfig` from the app generally

Reply:
Totally agree, there is already a task for it
https://github.com/iotaledger/firefly/issues/3319